package.json

```diff
@@ -1,93 +1,104 @@
   ],
   "scripts": {
-    "build": "ts-bridge --project tsconfig.build.json --clean",
-    "build:docs": "typedoc",
-    "lint": "yarn lint:eslint && yarn lint:constraints && yarn lint:misc --check && yarn lint:dependencies --check && yarn lint:changelog",
-    "lint:changelog": "auto-changelog validate --prettier",
-    "lint:constraints": "yarn constraints",
-    "lint:dependencies": "depcheck && yarn dedupe",
-    "lint:eslint": "eslint . --cache --ext js,cjs,ts",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:constraints --fix && yarn lint:misc --write && yarn lint:dependencies && yarn lint:changelog",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "build": "yarn ts-bridge --project tsconfig.build.json --verbose",
+    "build:clean": "rimraf dist '**/*.tsbuildinfo' && yarn build",
+    "build:docs": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run build:docs",
+    "build:types": "tsc --build tsconfig.build.json --verbose",
+    "changelog:update": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:update",
+    "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
+    "create-package": "ts-node scripts/create-package",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn constraints && yarn lint:dependencies",
+    "lint:dependencies": "depcheck && yarn dedupe --check",
+    "lint:dependencies:fix": "depcheck && yarn dedupe",
+    "lint:eslint": "eslint . --cache --ext js,cjs,mjs,ts",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix && yarn lint:dependencies:fix",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.old.md' '**/*.yml' '!.yarnrc.yml' '!merged-packages/**' --ignore-path .gitignore",
     "prepack": "./scripts/prepack.sh",
-    "test": "jest && jest-it-up && attw --pack",
-    "test:watch": "jest --watch"
+    "prepare-preview-builds": "./scripts/prepare-preview-builds.sh",
+    "publish-previews": "yarn workspaces foreach --all --no-private --parallel --verbose run publish:preview",
+    "setup": "yarn install",
+    "test": "yarn test:scripts --silent --collectCoverage=false --reporters=jest-silent-reporter && yarn test:packages",
+    "test:clean": "yarn workspaces foreach --all --parallel --verbose run test:clean && yarn test",
+    "test:packages": "yarn test:verbose --silent --collectCoverage=false --reporters=jest-silent-reporter",
+    "test:scripts": "yarn jest --config ./jest.config.scripts.js --silent",
+    "test:verbose": "yarn workspaces foreach --all --parallel --verbose run test:verbose",
+    "update-readme-content": "ts-node scripts/update-readme-content.ts",
+    "workspaces:list-versions": "./scripts/list-workspace-versions.sh"
+  },
+  "simple-git-hooks": {
+    "pre-push": "yarn lint"
+  },
```

Comment:
All these script should work with the exception of `prepare-preview-builds` and `publish-previews` which I have created an issue for because I still believe we will want this functionality in the near future https://github.com/MetaMask/metamask-design-system/issues/26